heat_and_trees/heat_and_trees.ipynb

```diff
@@ -595,8 +626,7 @@
     "thresholded_temp_p = (mean_temp_f.where(mean_temp_f > 90)\n",
     "    .hvplot(x='x', y='y', title='Mean Temp (F) > 90',\n",
     "            cmap=special_cmap, crs=crs, frame_width=400,\n",
-    "            frame_height=450,\n",
-    "            colorbar=False, legend=False)\n",
+    "            frame_height=450, legend=False)\n",
     "    .redim(value='Temperature (F)'))\n",
```

Comment:
It looks like `.redim(...)` has no effect? Maybe it's a regression. To be discussed during our meeting.

Reply:
You're right.  I'll remove it.